examples/grid.cpp

```diff
@@ -469,21 +501,126 @@ void DrmRenderer::Flipper::loop()
         m_lock.release();
         checkVaapiStatus(vaSyncSurface(m_display, id), "vaSyncSurface");
         bool late = waitingRenderTime();
+        m_fpsPlane->addFrame();
+
         m_lock.acquire();
         if (late) {
             ERROR("late m_fronts.size = %d", (int)m_fronts.size());
         }
         m_pending = true;
         if (m_fps)
         {
-          Timer t("flip");
+          //Timer t("flip");
```

Comment:
No, with this line, I will know the flip time when fps assigned. This line have functional. Not a forget remove one